.eslintrc.json

```diff
@@ -19,7 +19,7 @@
         "react"
     ],
     "rules": {
-        "no-unused-vars": "warn",
+        "no-unused-vars": ["warn", { "ignoreRestSiblings": true }],
```

Comment:
Everything looks good! What is this for?

Reply:
https://github.com/pmndrs/racing-game/pull/46/files#diff-1843bac69f436bba37a5448987963889810008090bde459ef5dbd50d46016ae7R4 

```js
const { directionLocal, axleLocal, chassisConnectionPointLocal, rollInfluence, ...filteredWheelInfo } = wheelInfo
```

We use the rest operator to exclude the properties we don't modify. Technically the properties we filter out are unused, but it's a functional way of not using them.